### ISSUE TITLE
Buying the microphone

### ISSUE THREAD

**Issue by MRo47** [nejohnson2/rpi-i2s#13]
@nejohnson2  thanks for sharing this project.
I would like to know the retailer you bought this breakout board from. These microphones seem impossible to solder without re flow ovens.
Regards

**Comment by nejohnson2** [comment]
Yeah...I had lots of trouble finding a good board.  I've ended up purchasing some ICS43434 boards from [notwire.co](https://www.notwired.co/products/detail/nwaudics43434-notwired-co/605570/).  I think the boards themselves are a little cumbersome with headers pre-soldered, but I made it work.  

Would love to hear of other sources if anyone finds one...

**Comment by hyansuper** [comment]
Do you know if the INMP441 module is supported? It has same pins 
https://h5.m.taobao.com/awp/core/detail.htm?id=583553653999&abtest=17&rn=7c3e84525d3f8eae44d1b5a0180b365e&sid=565e22e502cfc658ceb68404c2546aaf

**Comment by mrerdem** [comment]
I have got one from [https://www.tindie.com/products/atomic14/ics-43434-i2s-mems-microphone-breakout-board/?pt=ac_prod_search](https://www.tindie.com/products/atomic14/ics-43434-i2s-mems-microphone-breakout-board/?pt=ac_prod_search) with pretty cheap international shipping.
